templates/web/base/report/display.html

```diff
@@ -61,7 +61,7 @@
               c.uri_for( '/report/new', { longitude => longitude, latitude => latitude } )
           ) %]
     </p>
-[% ELSIF NOT shown_form %]
+[% ELSIF NOT shown_form AND NOT c.cobrand.call_hook('updates_disallowed', c, problem) %]
```

Comment:
No need to pass in `c` as mentioned elsewhere.
I wonder (untested) if `c.cobrand.updates_disallowed(problem)` works? As template just fails silently if methods don't exist, unlike code.